pkg/component/observability/opentelemetry/operator/crds.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package operator
+
+import (
+	_ "embed"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/component"
+	"github.com/gardener/gardener/pkg/component/crddeployer"
+)
+
+var (
+	//go:embed assets/crd-opentelemetry.io_opentelemetrycollectors.yaml
+	openTelemetryOpenTelemetryCollectorCRD string
+	//go:embed assets/crd-opentelemetry.io_instrumentations.yaml
+	openTelemetryInstrumentationCRD string
+	//go:embed assets/crd-opentelemetry.io_opampbridges.yaml
+	openTelemetryOpenTelemetryCollectorBridgeCRD string
```

Comment:
If we follow the pattern, this one should be
```suggestion
	openTelemetryOpAMPBridgeCRD string
```

Reply:
Thank you, seems like it was a typo on my part.